### ISSUE TITLE
Bolding/highlighting turned into ** and == when converting to PDF

### ISSUE THREAD

**Issue by HayaSat** [laurent22/joplin#5751]
I tried converting a document into PDF. The document contains lots of bolding and highlighting of text and some of these features are turned into **text** or ==text== for bolding and highlighting respectively. This makes it really inefficient to review my notes in PDF. 

## Environment

Joplin version: 2.5.12 
Platform: macOS
OS specifics: Catalina 10.15.7

## Steps to reproduce

1. Make notes with bold/highlights
2. convert note to pdf, or press mark down mode
3. bold are converted to **text** and highlights are ==text==

## Describe what you expected to happen
I thought all features would remain in PDF without any bugs

**Comment by laurent22** [comment]
Need and example of note that has this issue

**Comment by HayaSat** [comment]
Thanks for the reply
It seems like the problem fixed itself?
I'm not sure how, the problem persisted try after try but now it's working smoothly. 
May I update you on this thread should the same problem arise in the future?

Thanks

**Comment by laurent22** [comment]
Yes sure, and if it happens again please export the note as jex so that we can replicate the issue too.

**Comment by HayaSat** [comment]
Okay, so it's just one error but it happened again.

[Problem 1 Early roots of Psychology.pdf](https://github.com/laurent22/joplin/files/7566953/Problem.1.Early.roots.of.Psychology.pdf)

there is a ==text== on the final page. 

Jex file isn't supported to put in the comment. Is there another method you'd like me to resort to?

Thanks

**Comment by HayaSat** [comment]
Hi thank you for the notice. Yes, I have learned to fix the issue by
entering markdown and making sure that ** were correctly placed at the
front and behind the words.

Thank you so much and sorry for causing concern

haya
